lib/husky_musher/utils/redcap.py

```diff
@@ -70,9 +77,6 @@ def decorator(function):
     return decorator
 
 
-CACHE = FanoutCache(os.environ.get("CACHE"))
```

Comment:
I think this will require some changes to bin/dump-cache and bin/preheat-cache which currently reference CACHE directly.